indexer/packages/postgres/src/db/migrations/migration_files/20241002144813_change_affiliate_info_fee_columns.ts

```diff
@@ -0,0 +1,23 @@
+import * as Knex from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return knex
+    .schema
+    .alterTable('affiliate_info', (table) => {
+      table.dropColumn('totalReferredFees');
+      table.dropColumn('referredNetProtocolEarnings');
+      table.decimal('totalReferredTakerFees', null).notNullable().defaultTo(0);
+      table.decimal('totalReferredMakerFees', null).notNullable().defaultTo(0);
+    });
+}
+
+export async function down(knex: Knex): Promise<void> {
+  return knex
+    .schema
+    .alterTable('affiliate_info', (table) => {
+      table.decimal('totalReferredFees', null).notNullable().defaultTo(0).alter();
+      table.decimal('referredNetProtocolEarnings', null).notNullable().defaultTo(0).alter();
```

Comment:
totalReferredFees and referredNetProtocolEarnings dont exist in the table - so you shouldnt be able to alter them in this case. Should remove alter() here

Reply:
Good catch